contrib/systemd/containerized-installs/openvswitch.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+After=docker.service
+Requires=docker.service
+Before=origin-master.service
+
+[Service]
+EnvironmentFile=/etc/sysconfig/origin-node
+ExecStartPre=-/usr/bin/docker rm -f openvswitch
```

Comment:
Yeah, in most situations it wasn't being removed when we triggered the docker restart during SDN initialization or if the host were to reboot.